File: backend/src/main/java/com/pgvaale/backend/dto/DashboardStatsDTO.java
package com.pgvaale.backend.dto;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DashboardStatsDTO {
    private long totalUsers;
    private long totalOwners;
    private long totalTiffinProviders;
    private long totalMaids;
    private long totalPGs;
    private long pendingMaids;
    private long pendingTiffins;
    private long totalServiceProviders;
    private long totalAccounts;
    private BigDecimal averageFeedbackRating;

    // Getter
    public BigDecimal getAverageFeedbackRating() {
        return averageFeedbackRating;
    }
}

File: PGVaaleDotNetBackend/DTOs/DashboardStatsDTO.cs
namespace PGVaaleDotNetBackend.DTOs
{
    public class DashboardStatsDTO
    {
        public long TotalUsers { get; set; }
        public long TotalOwners { get; set; }
        public long TotalMaids { get; set; }
        public long TotalTiffins { get; set; }
        public long PendingMaids { get; set; }
        public long PendingTiffins { get; set; }
        public long ApprovedMaids { get; set; }
        public long ApprovedTiffins { get; set; }
        public long TotalPGs { get; set; }
        public long TotalBookings { get; set; }
        public long ActiveBookings { get; set; }
        public long CompletedBookings { get; set; }
        public double TotalRevenue { get; set; }
        public double MonthlyRevenue { get; set; }
    }
}
